radix-engine/tests/metering.rs

```diff
@@ -223,9 +223,9 @@ fn test_basic_transfer() {
         + 7000 /* read_owned_nodes */
         + 32500 /* read_substate */
         + 4000 /* run_native_method */
-        + 307263 /* run_wasm */
+        + 278748 /* run_wasm */
```

Comment:
Nice, that's not a small difference! Is this all attributed to this change?

Reply:
Yes, it's all form this PR. I guess string encoding, decoding and comparing are not cheap.